CMPUT301F24mohggg/MohgggDraw
# Us 02.05.01 organizer

- US 02.05.01: As an organizer, I want to notify chosen entrants.
- US 02.07.01: As an organizer, I want to send notifications to all entrants on the waiting list.
- US 02.07.02: As an organizer, I want to send notifications to all selected entrants.
- US 02.07.03: As an organizer, I want to send a notification to all canceled entrants.

**How to test it?**
    - Create a new event as an organizer, go back to homepage and to selected list to choose ur newly created event.
    - Go to firebase and add ur deviceId into any list inside that event u created
    - Click on select notification, go thru it, and select the list that u need to send notification to, and send it
    - u should receive notification both in android and in app notification.
